lua-tree/share/lua/5.1/kong/db/schema/metaschema.lua

```diff
@@ -79,6 +79,7 @@ local field_schema = {
   { type = { type = "string", one_of = keys(Schema.valid_types), required = true }, },
   { required = { type = "boolean" }, },
   { reference = { type = "string" }, },
+  { description = { type = "string" } },
```

Comment:
```suggestion
  { description = { type = "string", len_min = 10, len_max = 500}, },
```

https://github.com/Kong/kong-ee/blob/master/kong/db/schema/metaschema.lua#L82